### PR TITLE
Starting Afrikaans translation

### PR DESCRIPTION
**Pull request by Anri-Lombard**
Starting translation for issue #1164

### CODE REVIEW

**Comment by Anri-Lombard** [comment]
Progress update: ±10% of first draft done

**Comment by Anri-Lombard** [comment]
±30% of first draft done

**Comment by Anri-Lombard** [comment]
±40% of first draft done

**Comment by jwasham** [comment]
Looking great so far. Let me know when you want to merge.

**Comment by Anri-Lombard** [comment]
±50% of first draft done.

Thanks, John. My goal is to fix a few links in the table of contents (since vscode keeps reverting what I type up there for some reason) then to give you the thumbs up.

So:
Monday: ± 60% + fixed links, ready for merge from yourself.
Tuesday: ± 70%
Wednesday: ± 80%
Thursday: ± 90%
Friday: 100%, ready for another merge.
Saturday: Review by my Afrikaans friends and fixes - another merge where necessary.

**Comment by Anri-Lombard** [comment]
Hey John,

You can merge whenever your schedule allows.

**Comment by Anri-Lombard** [comment]
±60 of first draft done

File: README.md [hunk 1]
```diff
@@ -129,51 +129,62 @@ software development/engineering roles.
 
 ### Topics of Study
```
Comment (jwasham) [inline README.md:131]: Please remove the changes to the English readme.

File: translations/README-af.md [hunk 2]
```diff
@@ -0,0 +1,1200 @@
...
+            - [Distributed Hash Tables (video)](https://www.coursera.org/lecture/data-structures/distributed-hash-tables-tvH8H)
+
+    - [ ] Implement met 'n skikking en gebruik linear probing
+        - hash(k, m) - m is grootte van hash table
+        - add(sleutel, waarde) - as sleutel al klaar beskikbaar is, opdateer waarde
+        - exists(sleutel)
+        - get(sleutel)
+        - remove(sleutel)
+
+## Meer Kennis
```
Comment (jwasham) [inline translations/README-af.md:747]: I don't know Afrikaans, but it's looking great to me.

**Comment by Anri-Lombard** [comment]
Both files fixed 🙂

**Comment by Anri-Lombard** [comment]
±70 done of first draft - and I'm learning a ton along the way.

**Comment by jwasham** [comment]
Merged!

**Pull request merged**